_scripts/serve.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 rm -rf _site
-bundle exec jekyll serve --incremental --config _config.yml,_config-local.yml
+bundle exec jekyll serve --incremental --config _config.yml,_config-local.yml --host=0.0.0.0
```

Comment:
I think this should be left out. Listening on localhost only is a sensible default.